Ginger/GingerCore/Actions/ActSwitchWindow.cs

```diff
@@ -103,9 +103,19 @@ public Int32 WaitTime
             }
         }
 
+        private List<ePlatformType> obsoletePlatformList = new List<ePlatformType>()
```

Comment:
It is obsolete for all platforms for which it is supported. so why do we need to check?
We can return true no?